pkgs/development/libraries/librevisa/default.nix

```diff
@@ -24,7 +24,6 @@ stdenv.mkDerivation rec {
     '';
     homepage = http://www.librevisa.org/;
     license = licenses.gpl3Plus;
-    platforms = platforms.linux;
```

Comment:
This should be ```platforms.unix``` or ```platforms.linux ++ platforms.darwin``` to be more specific on what is supported.

Reply:
That's starting to get kinda prescriptive... do we really want someone trying to bring up a new platform like e.g. FreeBSD to have to go round adding their platform to _all packages_ they want to use?

Reply:
That's a valid concern. But at least hydra will default to "x86_64-linux" if no platforms is specified:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/release-lib.nix#L148

I think ```platforms.all``` makes the most sense here.

Reply:
Oh! Never would have guessed - ok.

Reply:
Done. Actually decided to go with `platforms.linux ++ platforms.darwin` in the end because actually _explicitly saying_ `platforms.all` feels like a bit of a lie. Like it's for something that has no platform dependence at all.